rps.js

```diff
@@ -0,0 +1,48 @@
+
+let cmChoice = Math.random();
+
+const game = function(user){
+if (cmChoice < 0.33) {
+    cmChoice = "rock";
+}
+else if( cmChoice > 0.66) { 
+    cmChoice = "scissors";
+}
+else {
+    cmChoice = "paper"
+}
+if (playGame) {

```

Comment:
you don't this since you get the use choice as parameter from the function call.